.travis.yml

```diff
@@ -1,31 +1,40 @@
 language: node_js
-node_js:
-  - "7"
-  - "6"
-  - "4"
-  - "0.12"
-  - "0.10"
-  - "iojs"
 
 compiler: gcc
 sudo: false
 
-os:
-  - linux
-  - osx
-
 env:
   global:
     - SKIP_SASS_BINARY_DOWNLOAD_FOR_CI=true
 
-matrix:
-  fast_finish: true
-  exclude:
-    - node_js: iojs
-    - node_js: "0.10"
+jobs:
+  include:
+    - stage: test
+      node_js: "7"
```

Comment:
Is it possible to use some kind of "latest" or "current" tag so we don't have to change this for node 8 etc..?